src/styles.scss

```diff
@@ -13,90 +17,475 @@ $blue: #007bff;
 $light-blue: #e5f2ff;
 $input-background: #f7f7fc;
 
-// Font colors.
 $light-font-color: #8d8f8f;
 $normal-font-color: #626480;
 $title-font-color: #0e0e23;
 $banner-font-color: #292941;
 $red-font-color: #ff3434;
 $green-font-color: #19b028;
 
+$text: #000000;
+$text-secondary: #545656;
+$text-muted: #999999;
+$landing-page-fc: #888;
+$landing-page-dark: #1b1c1e;
+$tile-bg: #ffffff;
+$tile-secondary-bg: #f4f6f6;
+$border: #dddddd;
+$highlight: #005bff;
+$highlight-secondary: #e5f2ff;
+$success: #2cc99a;
+$error: #ff6939;
+
+//-------------------------------
+// FONTS = Families
+//-------------------------------
+$font-primary: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+
 @font-face {
-  font-family: 'Eina01';
-  src: url('./assets/fonts/Eina01-Bold.ttf') format('truetype');
-  font-weight: bold;
+  font-family: 'color-emoji';
+  src: local('Apple Color Emoji'), local('Segoe UI Emoji'),
```

Comment:
we like to package up the fonts in our assets folder so we control everything for identity. fewer external touchpoints. Are you able to package any of these up as a ttf? I'm not a font expert and maybe this local stuff is ok?

Reply:
I actually deleted the Eina font, and this Emoji thing was already here. I'll clean this up at the end if it's not being used anywhere (I'll likely remove emojis anyways if I do spot any).